src/partials/cli-disclaimer.md

```diff
@@ -0,0 +1,3 @@
+{% info title="Before proceeding" %}
+Ensure you [**install**](/docs/tooling/command-line/installation#getting-started) the CLI, [**log in**](/docs/tooling/command-line/installation#getting-started) to your Appwrite account, and [**initialize**](/docs/tooling/command-line/installation#initialization) your Appwrite project.
```

Comment:
Shouldn't the 2nd one be `login`?

```suggestion
Ensure you [**install**](/docs/tooling/command-line/installation#getting-started) the CLI, [**log in**](/docs/tooling/command-line/installation#login) to your Appwrite account, and [**initialize**](/docs/tooling/command-line/installation#initialization) your Appwrite project.
```

Reply:
Yes, good catch. I caught that for the install link, but somehow missed it for login.